tests/test_helper/bats/bgp_data_plane.bats

```diff
@@ -0,0 +1,139 @@
+    wait_until "microovn_bgp_established $TEST_CONTAINER $vrf_device $BGP_PEER"
+
+    # create VIF that represents VM on the internal OVN network
+    local gw_lr="lr-$TEST_CONTAINER-microovn"
+    local guest_ls="ls-guest-net"
+    local guest_lsp_to_lr="lsp-to-gw"
+    local lrp_to_guest_ls="lrp-to-guest"
+    local guest_lrp_ip="192.168.10.1"
+    local guest_lrp_cidr="$guest_lrp_ip/24"
+    local guest_vm_ip="192.168.10.10"
+    local guest_vm_cidr="$guest_vm_ip/24"
+    local guest_vm_mac="02:00:aa:00:00:01"
+    local guest_vm_lsp="lsp-guest-vm"
+    local guest_vm_iface="guest-vm"
+    local guest_vm_ns="ns-guest"
+
+    echo "# ($TEST_CONTAINER) Create VIF in the OVN network that represents Virtual Machine with IP $guest_vm_cidr" >&3
+    echo "# ($TEST_CONTAINER) Create OVN network '$guest_ls' and connect it to router '$gw_lr' ($guest_lrp_cidr)"
+    lxc_exec "$TEST_CONTAINER" \
+        "microovn.ovn-nbctl \
+         -- \
+         lrp-add $gw_lr $lrp_to_guest_ls 02:00:ff:00:00:01 $guest_lrp_cidr \
+         -- \
+         ls-add $guest_ls \
+         -- \
+         lsp-add $guest_ls $guest_lsp_to_lr \
+         -- \
+         lsp-set-type $guest_lsp_to_lr router \
+         -- \
+         lsp-set-options $guest_lsp_to_lr router-port=$lrp_to_guest_ls \
+         -- \
+         lsp-set-addresses $guest_lsp_to_lr router \
+         "
+
+    echo "# ($TEST_CONTAINER) Create LSP '$guest_vm_lsp' ($guest_vm_cidr)"
+    lxc_exec "$TEST_CONTAINER" \
+        "microovn.ovn-nbctl \
+         -- \
+         lsp-add $guest_ls $guest_vm_lsp \
+         -- \
+         lsp-set-addresses $guest_vm_lsp '$guest_vm_mac $guest_vm_cidr'"
+
+    echo "# ($TEST_CONTAINER) Create netns '$guest_vm_ns' and move VM interface '$guest_vm_iface' into it"
+    lxc_exec "$TEST_CONTAINER" "ip netns add $guest_vm_ns"
+    lxc_exec "$TEST_CONTAINER" \
+        "microovn.ovs-vsctl \
+         -- \
+         add-port br-int $guest_vm_iface \
+         -- \
+         set Interface $guest_vm_iface type=internal external_ids:iface-id=$guest_vm_lsp"
+    netns_ifadd "$TEST_CONTAINER" "$guest_vm_ns" "$guest_vm_iface" "$guest_vm_mac" "$guest_vm_cidr"
+    wait_until "microovn_lsp_up $TEST_CONTAINER $guest_vm_lsp"
+
+    echo "# ($TEST_CONTAINER) Set VM's default route via $guest_lrp_ip"
+    lxc_exec "$TEST_CONTAINER" "ip netns exec $guest_vm_ns ip route add default via $guest_lrp_ip"
+
+    # Note (mkalcok): Until OVN routers are capable of learning routes advertised
+    # by their BGP neighbors, we need to use a crutch in a form static default route
+    # via the BGP neighbor's IPv6 LLA
+    #
+    # Note 2 (mkalcok): It also seems that without specifying egress port for the route, the returning
+    # traffic seems to be a hit-or-miss.
+    local neighbor_lla
+    local egress_port="lrp-$TEST_CONTAINER-$OVN_CONTAINER_INT_IFACE"
+    neighbor_lla=$(microovn_bgp_neighbor_address "$TEST_CONTAINER" "$vrf_device" "${OVN_CONTAINER_INT_IFACE}-bgp")
```

Comment:
We need the IP of the neighbor, not the local port, so the `ovn-nbctl` isn't an option.
While true that we could parse output of the `ip a show` on the `BGP_PEER` host, I find the output format bit hard to process. I was also thinking about future test scenarios with multiple BGP peers per single chassis, this approach will let you find out addr of your neighbors from the host itself, instead of having to figure out which host (container) is the BGP neighbor for the particular chassis.